lldb/source/Target/Process.cpp

```diff
@@ -2545,6 +2546,11 @@ ModuleSP Process::ReadModuleFromMemory(const FileSpec &file_spec,
   ModuleSP module_sp(new Module(file_spec, ArchSpec()));
   if (module_sp) {
     Status error;
+    std::unique_ptr<Progress> progress_up;
+    if (!GetCoreFile())
```

Comment:
Might be better to use the bool accessor function:
```
if (IsLiveDebugSession())
```

Reply:
The name is a little misleading, `IsLiveDebugSession` is only false for `PostMortemProcess`.  Maybe we should have a `Process::IsCorefileProcess` or something, or I could put a comment there about what I'm doing.

Reply:
`PostMortemProcess` is what all of the core file processes (elf, mach-o and minidump) inherit from. That way all core file processes don't forget to override this function. I agree the name isn't the most clear.

Reply:
Oh whoops, my bad on that one, thanks.  I'll change to use this instead, it's clearer.